news/services/refiner.py

```
@@ -118,3 +118,86 @@ def get_refined_body(self, raw_data):
+                },
+            ]
+
+            response = self.client.models.generate_content(
+                model=self.model_name,
+                contents=prompt,
+                config={
+                    "safety_settings": safety_settings,
+                },
+            )
+            return response.text.strip()
+        except Exception as e:
```

Ruff (0.14.13)

Do not catch blind exception: `Exception`

(BLE001)